CHANGELOG.md

```diff
@@ -32,7 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking Changes
 
-- Bump MSRV to 1.63.0
+- Bump MSRV to 1.64.0
```

Comment:
It'd be nice if we had an idea of what was pinning the MSRV high. It's better if the MSRV is lower, cause then implementors can have a lower MSRV and it's compatible with more versions of Rust.

Reply:
In this case `gethostname` wanted an MSRV bump.  With an older version of `gethosname` it could possibly be pushed lower.